qiskit/pulse/commands/persistent_value.py

```diff
@@ -61,8 +61,7 @@ def __eq__(self, other: 'PersistentValue'):
         Returns:
             bool: are self and other equal
         """
-        if type(self) is type(other) and \
-                self.value == other.value:
+        if super().__eq__(other) and self.value == other.value:
```

Comment:
isn't this redundant? will `self.value` be checked in the `self.command == other.command` check?

Reply:
No, since this `value` is specific to the `PersistentValue` command.